x-pack/plugin/enrich/qa/rest/roles.yml

```diff
@@ -0,0 +1,26 @@
+enrich:
+  cluster:
+    - manage_enrich
+    - manage_ingest_pipelines
+    - monitor
+  indices:
+    - names: [ '.enrich-my_policy*', 'my-index', 'my-source-index' ]
+      privileges:
+        - manage
+        - monitor
+        - read
+        - write
+        - create_index
```

Comment:
I think `create_index ` and `monitor` privileges are not needed, because these are part of the `management` privilege.

Reply:
added as part of the newest commit to move test